docs/tutorials/notebooks/Unbalanced_OT.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<p>... problem using the {class}<code>~ott.solvers.linear.sinkhorn.Sinkhorn</code> algorithm.</p><p>Instead of ..., OTT's solver uses (here you can also link if you want).</p><p>... and solving the corresponding balanced {class}<code>~ott.problems.linear.linear_problem.LinearProblem</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>